app/views/shared/_topbanner.html.erb

```diff
@@ -1,7 +1,8 @@
 <section>
   <div class="container-fluid">
-    <div class="jumbotron" align="middle">
-      <img src="../../assets/images/top-banner-logo.png" height="125" alt="Top banner Taste of Code" />
+    <div class="container" align="middle">
```

Comment:
something Anika is going to fix as well, since the container is going to be in the main html pages